test/fail_compilation/fail_arrayop3a.d

```diff
@@ -1,23 +1,14 @@
-// REQUIRED_ARGS: -o-
-
 /*
-example output from druntime
+REQUIRED_ARGS: -o-
+TEST_OUTPUT:
 ----
-../../druntime/import/core/internal/arrayop.d(160): Error: static assert  "Binary op `*` not supported for element type X."
```

Comment:
Note that this _output_ is just an informal comment and was never enforced by the test suite.
Ditto for the other `fail_arrayop` tests.